types/begin__data/tsconfig.json

```diff
@@ -0,0 +1,20 @@
+{
+    "compilerOptions": {
+        "module": "commonjs",
+        "lib": ["es2018"],
+        "noImplicitAny": true,
+        "noImplicitThis": true,
+        "strictNullChecks": true,
+        "strictFunctionTypes": true,
+        "esModuleInterop": true,
```

Comment:
```suggestion
```
it's not (usually) used (or recommended) on DT and the package looks to be plain, old CJS:
https://github.com/beginner-corp/begin-data/blob/master/package.json#L5
https://github.com/beginner-corp/begin-data/blob/master/index.js#L9

Reply:
Thank you! I've changed to use `import {set, get, ...} from ...` instead with this change.